cmd/minikube/cmd/tunnel.go

```diff
@@ -116,6 +122,38 @@ var tunnelCmd = &cobra.Command{
 	},
 }
 
+func cleanupLock() {
+	if lockHandle != nil {
+		err := lockHandle.Unlock()
+		if err != nil {
+			out.Styled(style.Warning, fmt.Sprintf("failed to release lock during cleanup: %v", err))
+		}
+	}
+}
+
+func mustLockOrExit() {
+	tunnelLockPath := filepath.Join(localpath.MiniPath(), ".tunnel_lock")
+
+	_, err := os.OpenFile(tunnelLockPath, os.O_RDWR|os.O_TRUNC, 0600)
+	if err != nil {
+		if !os.IsNotExist(err) {
+			exit.Error(reason.SvcTunnelStart, fmt.Sprintf("error opening lock file for tunnel file (%s)", tunnelLockPath), err)
+		}
+		_, err = os.Create(tunnelLockPath)
+		if err != nil {
+			exit.Error(reason.SvcTunnelStart, fmt.Sprintf("error creating lock for tunnel file (%s)", tunnelLockPath), err)
+		}
+	}
```

Comment:
```suggestion
```

Reply:
Thank you so much for debugging it. I have made changes as suggested. I also confirmed source code of the package to ensure they file is created upon `TryLock()` across platforms (e..g, for *nix [here](https://github.com/juju/fslock/blob/4d5c94c67b4b207e1ab4ebca6b4e47f174618b86/fslock_nix.go#L36) and [here](https://github.com/juju/fslock/blob/4d5c94c67b4b207e1ab4ebca6b4e47f174618b86/fslock_nix.go#L49)).